packages/flutter_tools/test/integration.shard/vmservice_integration_test.dart

```diff
@@ -104,6 +104,26 @@ void main() {
         }
       );
       expect(updateResponse.json['value'], 'Brightness.dark');
+
+      // Change the brightness back to light
+      final Response verifyResponse = await vmService.callServiceExtension(
+        'ext.flutter.brightnessOverride',
+        isolateId: isolate.id,
+        args: <String, String>{
+          'value': 'Brightness.light',
+        }
+      );
+      expect(verifyResponse.json['value'], 'Brightness.light');
+
+      // Change the brightness back to light
+      final Response bogusResponse = await vmService.callServiceExtension(
+        'ext.flutter.brightnessOverride',
+        isolateId: isolate.id,
+        args: <String, String>{
+          'value': 'dark',
+        }
```

Comment:
Can you add a comment that this is purposely bogus?  It looks like a bug in the test.

Reply:
Done

Reply:
Done